tuned_lens/plotting/prediction_trajectory.py

```diff
@@ -538,6 +542,42 @@ def cross_entropy(self, **kwargs) -> TrajectoryStatistic:
             stats=stats,
         )
 
+    def rank(self, show_ranks=False, **kwargs) -> TrajectoryStatistic:
+        """The rank of the targets among the predictions.
+
+        Args:
+            show_ranks: Whether to show the the rank of the target or the top token.
```

Comment:
I don't understand this, isn't the rank of the top token always 1?  Or you mean that usually the number output is the id of the top token, and this makes it output the rank instead?

maybe then:
```suggestion
            show_ranks: If true, show the rank of the target; instead of showing the id of the top (rank-1) token.
```

Reply:
The latter. It actually outputs the token string itself by default.